python/cudf/cudf/tests/pytest.ini

```diff
@@ -8,3 +8,4 @@ filterwarnings =
     error
     ignore:::.*xdist.*
     ignore:::.*pytest.*
+    ignore:Passing a BlockManager to DataFrame is deprecated:DeprecationWarning
```

Comment:
In a follow up, could you add a comment here that this comes from pyarrow starting with pandas 2.2?

Reply:
Done, in this pr itself 👍